### ISSUE TITLE
The "WriteVersionInfoToBuildLog" task could not be loaded from the assembly

### ISSUE THREAD

**Issue by KiLLeRRaT** [GitTools/GitVersion#2153]
Build Issue after installing `5.1.4-beta1.216`

When I install a fresh copy of GitVersionTask on a .NET Framework 4.7.2 project, I get the following error when trying to build

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		The "WriteVersionInfoToBuildLog" task could not be loaded from the assembly C:\ProjectsG.Git\Gouws.WakeOnLan\packages\GitVersionTask.5.1.4-beta1.216\build\..\tools\net472\GitVersionTask.MsBuild.dll. Could not load file or assembly 'System.Runtime.Loader, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.	Gouws.WakeOnLan	C:\ProjectsG.Git\Gouws.WakeOnLan\packages\GitVersionTask.5.1.4-beta1.216\build\GitVersionTask.targets	10	
```

Any ideas? I have been regularly trying the latest pre-release versions because I'm after some functionality in there, but each time I'm hit with a build error.

Cheers,

**Comment by arturcic** [comment]
I guess this one is a duplicate of #2131

**Comment by arturcic** [comment]
@KiLLeRRaT can you confirm it's fixed in v5.1.4-beta1-220?

**Comment by KiLLeRRaT** [comment]
> @KiLLeRRaT can you confirm it's fixed in v5.1.4-beta1-220?

Hi @arturcic , yes I have tested it and it has been fixed. 

Thanks for the effort in sorting it out!
